### PR TITLE
Queue dashboard

### PR DESCRIPTION
**Pull request by aginzberg**
Allows the user to select multiple tasks from their dashboard to go back and finish. Lot of moving parts here so I will continue to test tomorrow morning. Would appreciate comments/issues if anyone has time.

### CODE REVIEW

**Comment by maniksingh92** [comment]
@aginzberg could you describe this PR a little more in detail i.e what to expect in UI, so that we know exactly how to go about testing it

**Comment by aginzberg** [comment]
@maniksingh92 basically you can now select multiple in progress tasks from dashboard and click the go to finish them icon. then you have the option to save, skip, or submit these tasks one at a time in a queue consisting only of the tasks that you selected. does that help?

**Comment by dmorina** [comment]
lgtm

File: static/js/dashboard/controllers/dashboard.controller.js [hunk 1]
```diff
@@ -89,11 +89,8 @@
 
       //TODO process data as html upon click of inprogress task and allow worker to finish/delete task
```
Comment (nistala) [inline static/js/dashboard/controllers/dashboard.controller.js]: remove todo?

File: static/js/dashboard/services/dashboard.service.js [hunk 2]
```diff
@@ -27,6 +27,8 @@
     };
     return Dashboard;
 
+    var savedQueue = [];
```
Comment (nistala) [inline static/js/dashboard/services/dashboard.service.js]: should be ok to delete this

**Comment by dmorina** [comment]
should be good to merge @nistala

**Comment by maniksingh92** [comment]
@aginzberg okay. lgtm

**Comment by mbernst** [comment]
Procedure question: @nistala had unaddressed requests. Did this go off into
Slack for the lgtm before merging? Or decision made not to address them?

**Comment by aginzberg** [comment]
In this case, I believe @nistala's comments were addressed before the merge.

**Comment by mbernst** [comment]
OK thanks

**Pull request merged**